automatic/conan/tools/chocolateyInstall.ps1

```diff
@@ -22,4 +22,4 @@ Install-ChocolateyPackage @packageArgs
 
 # Add conan to PATH
 $conan_path = "C:\Program Files\Conan\conan"
-Install-ChocolateyPath -PathToInstall $conan_path -PathType User
+Install-ChocolateyPath -PathToInstall $conan_path -PathType Process

```

Comment:
```suggestion
Install-ChocolateyPath -PathToInstall $conan_path -PathType Machine
```
`Process` would mean only the environment of the current executing process and would mean that `conan` isn't available for anyone.